tools/deploy/mysql_test/test_suite/issue1365/issue1365.test

```diff
@@ -0,0 +1,20 @@
+use test;
+DROP TABLE IF EXISTS numeric_table;
+create table IF NOT EXISTS numeric_table ( col1 INTEGER comment 'INTEGER', col2 INT comment 'INT', col3 SMALLINT, col4 TINYINT, col5 MEDIUMINT, col6 BIGINT, col7 DECIMAL comment 'DECIMAL', col8 NUMERIC comment 'NUMERIC', col9 FLOAT, col10 DOUBLE, col11 BIT ) DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_bin;
+SELECT `TABLE_NAME`, `COLUMN_NAME`, `COLUMN_TYPE`, `COLUMN_COMMENT`,`IS_NULLABLE`, `COLUMN_KEY`, `COLUMN_DEFAULT`, `EXTRA`, `CHARACTER_SET_NAME`,`COLLATION_NAME`, NULL AS `GENERATION_EXPRESSION` FROM`INFORMATION_SCHEMA`.`COLUMNS` WHERE `TABLE_SCHEMA` = 'test' AND `TABLE_NAME`= 'numeric_table' ORDER BY `ORDINAL_POSITION`;
```

Comment:
1. 'issue1364.test' should be rename as 'bug_issue1365.test'
2. the .test file should move to a suitable mysqltest directory, I suggest 'tools/deploy/mysql_test/test_suite/information_schema/t' , the last 't' should be noticed, and use case '--'information_schema.bug_issue1365' to run and it automatically generate the result file in  'tools/deploy/mysql_test/test_suite/information_schema/r'